sample/ios/iosApp/sample/SampleTextAppearanceResourceProvider.swift

```diff
@@ -2,24 +2,28 @@ import Trikot_viewmodels
 import ViewModelsSample
 
 class SampleTextAppearanceResourceProvider: TextAppearanceViewModelResourceProvider {
-    func textAppearance(fromResource resource: TextAppearanceResource) -> TextAppearanceAttributes? {
+    func textAppearance(fromResource resource: TextAppearanceResource) -> TextAppearanceAttributes {
         if let sampleResource = resource as? SampleTextAppearanceResource {
             switch sampleResource {
             case .textAppearanceBold:
-                return TextAppearanceAttributes(font: .boldSystemFont(ofSize: 12), foregroundColor: .black)
+                return TextAppearanceAttributes(attributes: [.font: UIFont.boldSystemFont(ofSize: 12), .foregroundColor: UIColor.black])
```

Comment:
Why having an explicit `foregroundColor` instead of supplying `NSAttributedString.Key.foregroundColor` in the `attributes` parameter?

Reply:
A bit cross eyed this morning, `foregroundColor` is part of the array 🤦